pkg/virt-controller/watch/vmi.go

```diff
@@ -2100,3 +2109,25 @@ func (c *VMIController) getVolumePhaseMessageReason(volume *virtv1.Volume, names
 	}
 	return virtv1.VolumePending, PVCNotReadyReason, "PVC is in phase Lost"
 }
+
+func (c *VMIController) handleDynamicInterfaceRequests(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) error {
+	podAnnotations := pod.GetAnnotations()
+	log.Log.Object(pod).V(4).Infof("current multus annotation for pod: %s", podAnnotations[networkv1.NetworkAttachmentAnnot])
```

Comment:
can you please place this info in the last logging? If for some reason you need this info in the error, you can add it there before returning.